root/io/io/TBufferMergerBenchmarks.cxx

```diff
@@ -45,18 +51,21 @@ static void BM_TBufferFile_GetFile(benchmark::State &state)
 }
 BENCHMARK(BM_TBufferFile_GetFile)->Unit(benchmark::kMicrosecond);
 BENCHMARK(BM_TBufferFile_GetFile)->Unit(benchmark::kMicrosecond)->UseRealTime()->ThreadPerCpu();
-BENCHMARK(BM_TBufferFile_GetFile)->Unit(benchmark::kMicrosecond)->UseRealTime()->ThreadRange(1, 256);
+BENCHMARK(BM_TBufferFile_GetFile)
+   ->Unit(benchmark::kMicrosecond)
+   ->UseRealTime()
+   ->ThreadRange(1, GetNumberHardwareThreads() * 4);
 
 /// Creates a TMemFile, fills a TTree with random numbers. The data is written if it exceeds 32MB.
-inline void FillTreeWithRandomData(TBufferMerger &merger, size_t nEntriesPerWorker = 24 * 1024, int flush = 32 )
+inline void FillTreeWithRandomData(TBufferMerger &merger, size_t nEntriesPerWorker = 24 * 1024, int flush = 32)
 {
    thread_local std::default_random_engine g;
    std::uniform_real_distribution<double> dist(0.0, 1.0);
 
    auto f = merger.GetFile();
    auto t = new TTree("random", "random");
    t->ResetBit(kMustCleanup);
-   t->SetAutoFlush(-(flush) * 1024 * 1024); // Flush at exceeding 32MB
```

Comment:
This change seems redundant.

Reply:
strange, I didn't recheck after clang-format, it is because of "pointer-like" behavior?